libvast/vast/fbs/coder.fbs

```diff
@@ -0,0 +1,40 @@
+include "bitmap.fbs";
+
+namespace vast.fbs.coder.detail;
+
+table Base {
+  values: [ulong] (required);
```

Comment:
This should be `[ubyte]` so we don't store excess bytes for every value if `sizeof(coder::value_type) < sizeof(uint64_t)`.

Reply:
No, `[ulong]` is correct. A "base" is a part of the arithmetic decomposition of a value.